docs/integrate/risingwave/index.md

```diff
@@ -0,0 +1,99 @@
+(risingwave)=
+
+# RisingWave
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://www.risingwave.com/_next/static/media/risingwave-logo-black-text.11ccd229.svg){w=180px}](https://www.risingwave.com/)
+```
+
+[RisingWave] is a stream processing and management platform that allows
+configuring data sources, views on that data, and destinations where
+results are materialized.
+It provides both a Postgres-compatible SQL interface, like CrateDB,
+and a DataFrame-style Python interface.
+
+RisingWave can ingest millions of events per second, continuously join
+and analyze live streams with historical data, serve ad-hoc queries at
```

Comment:
> with historical data

Not sure about this

Reply:
> serve ad-hoc queries

Or this

Reply:
I've copied it from their presentations somewhere, most probably the homepage and the GitHub README, but sure we can tune it down.